### PR TITLE
Improve information for Picroft and Desktop user

### PR DESCRIPTION
**Pull request by forslund**
Inform about raspotify for Picroft and to use the desktop spotify player for desktop users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,31 @@
...
+Stream your favorite music from the popular Spotify music service. Spotify
 Premium users can search and play tracks from their own playlists or the huge
 Spotify music library.
 
 You can also control your Mycroft device using the Spotify Connect system.
 So play DJ on your phone while listening on Mycroft!
 
+### This skill doesn't do any playback
+
+This skills works with the spotify connect protocol and generally requires the user to setup a spotify player themselves.
```
Comment (joshuajaharwood) [inline README.md]:
```suggestion
This skill works with the Spotify Connect protocol to interact with Spotify devices, but doesn't perform any playback itself. If you want playback on the hosting Mycroft device, you'll need to set up a player yourself.
```

File: README.md [hunk 2]
```diff
@@ -2,13 +2,31 @@
...
+
+For picroft users [raspotify](https://github.com/dtcooper/raspotify) is a good choice.
```
Comment (joshuajaharwood) [inline README.md]:
```suggestion
For Picroft users, [raspotify](https://github.com/dtcooper/raspotify) is a good choice.
```

Comment (joshuajaharwood) [reply README.md]: It's worth mentioning that raspotify on its own runs on a seperate user to the pi user, and so can't access Picroft's Pulseaudio instance without config changes. I think we already discussed this once, so sorry if I'm repeating myself :)

File: README.md [hunk 3]
```diff
@@ -2,13 +2,31 @@
...
+- DEVICE_NAME to the name of the Mycroft unit (as registered at home.mycroft.ai):
+`DEVICE_NAME="<My Mycroft Unit>"
+
+- and username and password
+`OPTIONS="--username <My Username> --password <My Password>"`
+
+For desktop users the official spotify player works well.
+
+The exception to this is the Mark-1 which is shipped with a spotify player library.
+
```
Comment (joshuajaharwood) [inline README.md]:
```suggestion

Install it and then make changes to `/etc/default/raspotify.conf` as follows:

- Set `DEVICE_NAME` to the name of the Mycroft unit (as registered on the [Mycroft website](https://account.mycroft.ai/devices)): 
  
  `DEVICE_NAME="<My Mycroft Unit>"`

- Set your Spotify username and password under `OPTIONS`:

  `OPTIONS="--username <My Username> --password <My Password>"`

For desktop users the official Spotify player works well.

The exception to this is the Mark-1 which is shipped with a Spotify player library.
```

Comment (joshuajaharwood) [reply README.md]: I don't think it's explicitly necessary to set the DEVICE_NAME to the Mycroft name verbatim, but I can understand doing it for the sake of simplicity.

**Review by joshuajaharwood — CHANGES_REQUESTED** [review]
Sorry, I did some of these as comments rather than a review - my first PR review. Hope these help!

File: README.md (hunk 2 above)
Comment (fermulator) [reply README.md]: which "config changes" are you remembering here? - I don't think I recall anything special/extra about it.
(once I understood that raspotify was needed)

File: README.md (hunk 1 above)
Comment (fermulator) [reply README.md]: Ya this is an important aspect to consider.  Users don't HAVE to install raspotify locally, if they don't want to playback on the mycroft device itself, and rather just control other Spotify compatible players on the network.

File: README.md (hunk 3 above)
Comment (fermulator) [reply README.md]: Could just say "it is recommended to ...", (it helps with device identification of course)

File: README.md (hunk 2 above)
Comment (joshuajaharwood) [reply README.md]: I edited the service to run under the pi user, rather than the raspotify user.

**Comment by forslund** [comment]
Thanks for all the feedback! I'll revise the text based on this

**Pull request merged**